config/settings.yml

```diff
@@ -800,19 +735,11 @@
       - POD_SCHEDULED
       - POD_FAILEDSCHEDULING
       :detail:
-      - DatacenterRenamedEvent
-      - GeneralUserEvent
-      - LicenseServerUnavailableEvent
-      - Rename_Task
       - Rename_Task_Complete
```

Comment:
These complete events are added [here](https://github.com/ManageIQ/manageiq/blob/f15a504cb08f09cb82ce43cd5664253690af75a3/app/models/ems_event.rb#L89) if they are one of the [task final events](https://github.com/ManageIQ/manageiq/blob/f15a504cb08f09cb82ce43cd5664253690af75a3/config/settings.yml#L873). 
I think these task final events are all VMware related. @Fryguy Can you confirm?